### DISCUSSION TITLE
edit-tools 对单标签边进行文本编辑出错

### DISCUSSION THREAD

**Discussion by avrinfly** [antvis/X6#1881] [Q&A]
![image](https://user-images.githubusercontent.com/30068005/157203267-e0ce333f-5123-4ba2-b327-5bf573b142a8.png)
如何利用边工具，针对单标签的边进行文本编辑，而不是创建新的边。这个setText我看有优化，但具体怎么使用，有大佬指教一下吗？

**Comment by NewByVector** [comment]
有复现 demo 看下吗

**Reply by avrinfly** [reply]
> 有复现 demo 看下吗

https://codesandbox.io/s/green-sun-esvswc

现在双击关系线的非文本位置，也会出现输入框。
期望是：最好是关系线的非文本位置点击无效，或者至少不要在点击关系线的非文本位置出现输入框

**Reply by NewByVector** [reply]
可以在 edge-editor 工具上增加一个配置，控制点击非文本位置是否新建 label。欢迎提 PR。
